### PR TITLE
fix(fwc): Fixes the Master Warning on startup

### PR DESCRIPTION
**Pull request by Maximilian-Reuter**
Fixes #[issue_no]

## Summary of Changes

On startup in C/D the aircraft plays a master warning until the startup EWD messages are empty. This was caused by the pseudo FWC checking whether the aircraft was on ground only after checking for dual engine failure which meant that the FWC didn't update the on ground status until after the check for dual engine failure and since the on ground status was initialized with false, the dual eng failure triggered on initial load.

This PR changes the order of those checks so that the setting of the on ground status happens first.

## Additional context

Discord username (if different from GitHub): \_chaoz_

## Testing instructions

just a quick fix that shouldn't affect anything beyond the fwc initialization.

1. start C/D
2. power up the aircraft
3. no master warning should occur

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: fbw-a32nx/src/systems/instruments/src/EWD/PseudoFWC.ts [hunk 1]
```diff
@@ -1594,9 +1597,11 @@ export class PseudoFWC {
                     }
 
                     if (value.failure === 3) {
+                        console.log(`Playing Master warning Source:${key}`);
```
Comment (Saschl) [inline fbw-a32nx/src/systems/instruments/src/EWD/PseudoFWC.ts:1600]: Pls remove the logs and we're good to go :)

**Comment by Saschl** [comment]
Also might be worth a changelog entry I think. Would go to 0.11 too imo

**Comment by tracernz** [comment]
> Also might be worth a changelog entry I think. Would go to 0.11 too imo

I’d suggest keeping all the master changelogs under 0.12 and once the 0.11 release is done tidy the cherry-picked ones up all at once.

**Comment by Maximilian-Reuter** [comment]
> ~Also might be worth a changelog entry I think. Would go to 0.11 too imo~ EDIT: Tracers suggestion is better

is this even an issue on stable right now ?

File: fbw-a32nx/src/systems/instruments/src/EWD/PseudoFWC.ts (hunk 1 above)
Comment (Maximilian-Reuter) [reply fbw-a32nx/src/systems/instruments/src/EWD/PseudoFWC.ts:1600]: duh, knew I forgot smth

**Comment by tracernz** [comment]
> > ~Also might be worth a changelog entry I think. Would go to 0.11 too imo~ EDIT: Tracers suggestion is better
> 
> is this even an issue on stable right now ?

0.11 stable is master as of 2 days ago.

**Review by Saschl — APPROVED** [review]

**Comment by Saschl** [comment]
QA Report

Discord: saschl
Object of testing:
Tier of Testing : 1
Date : 26/10/2023

Testing Process:

Loaded the plane and powered up the plane. No master warning audible.
Slewed into the air with engines off. Master warning comes up as expected.

Testing Results:
Passed

**Pull request merged**